webapp/src/routes/BlockProducers/index.js

```diff
@@ -60,11 +63,14 @@ PaginationWrapper.propTypes = {
   totalPages: PropTypes.number,
   page: PropTypes.number,
   onPageChange: PropTypes.func,
-  loading: PropTypes.bool,
-  chipFilter: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+  loading: PropTypes.bool
 }
 
 const Producers = () => {
+  const chips = CHIPS_NAMES.map((e) => {
```

Comment:
we can do that as a single line function

example: 
` const chips = CHIPS_NAMES.map((e) => ({ name: e }))`